smoke-tests/fake-backend/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-bin.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
Please add checksum as in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/899263612d338a85310eec4e3c54aaac32aef327/gradle/wrapper/gradle-wrapper.properties#L6

Reply:
I see none of our smoke tests projects do so far. Do you mind if I go ahead and follow up with a PR to remove the "subrepos" to use our normal Gradle? While working on this I realized

1) It's a bit (not dealbreaker) annoying to start up another intellij to tweak each project
2) It'd be nice to apply our normal gradle scripts for spotless, license, etc
3) Worried about interaction with tooling, such as #1133 

I could either flag out these projects but since the normal build would only run `compileJava` essentially, I think it's ok to actually just have them enabled by default. What do you think?

Reply:
I have moved them outside our usual Gradle structure on purpose. E.g. `play` project contributes a lot of Gradle warnings during the build. I did not want them to pollute all our builds.

Reply:
Hmm - ok I've added the distribution sums for now. But I think the tradeoffs of the separate projects warrant another look into what can be done about things like the deprecation warnings.